docs/scripts.rst

```diff
@@ -58,7 +58,7 @@ table.
   Add Table? (y/N): y
   table-name: species
   table-url: http://esapubs.org/archive/ecol/E084/093/Mammal_lifehistories_v2.txt
-  missingValues (separated by ';'):
+  missing-values (separated by ';'):
```

Comment:
According to a test run `retriever  new_json`, this looks like it should be `missing values` instead of `missing-values`, so I guess we can go a head and update that correctly.